code/game/gamemodes/wizard/artefact.dm

```diff
@@ -880,3 +882,20 @@ var/global/list/multiverse = list()
 	heal_brute = 25
 	heal_burn = 25
 	heal_oxy = 25
+
+/mob/living/proc/skeleton_reset()
+	set name = "Offer skeleton to ghosts"
+	set category = "Skeleton"
+	set desc = "Give up your skeleton to ghosts so you can observe the round."
```

Comment:
This is like allowing converted players to opt out of their role. I do not support this.

Reply:
I mean...we already do? You can ahelp if you get converted and admins probably offer your body to ghosts. I don't think admins usually turn that down, or do they?

Reply:
I'd personally agree with TDSSS on this, I would rather have someone interested in being a convertee instead of someone being salty and halfassing it because they're unhappy with how things went down

Reply:
Then that should be up to the admins.

I don't want to encourage behavior like that. It's awful and terrible, and quite frankly, if you can't put up with it, then...why are you playing SS13 again?

Reply:
Isn't this more off-sourcing an admin task to something more automatic though? I would imagine if something could be automated it would

Reply:
Imagine suiciding just after calling this proc 🤔

Reply:
Would that basically give control of a then-dead skeleton mob? That's bad. Can that be prevented?

Reply:
I can't see any checks for being dead or incapacitated, so yeah I think that should be changed

Reply:
I have to go with Couls, TDSSS, and taukausanke on this. It allows people who don't want to play a specific role (which is something we already have as a preference system) being able to not. Their loss of being able to play. It also would allow for an override of someone who died and gets salty because of their death to not take the role and help their killer but still gives the killer the opportunity to use the body, as well as the opportunity of a dead body of a person that also left the game shortly after death to be used more easily. We cannot and should not force someone to play something they don't want to. We all play SS13 for various reasons, and even I dislike most antag roles. Most of the time, I don't even try to play antag (don't role for most, don't like most).